app/denom.go

```diff
@@ -0,0 +1,12 @@
+package app
```

Comment:
Personally, I'd say this file is too small to constitute a new file but it's a slight preference

Reply:
hmm I'm not sure how to proceed. I'm trying to extract stuff from app.go because it's massive. IMO the denomination constants don't need to be located in the same file as the rest of the app set up logic.

If this doesn't constitute a new file, where should I put it? I don't think reverting and putting it back in app.go makes sense.

Reply:
I guess I could rename this file consts.go and put other constants defined in the `package app` in here? But calling it `consts.go` now felt premature

Reply:
Going to merge for now b/c we can easily change this in a follow up PR.